### ISSUE TITLE
5.3.1 交叉验证

### ISSUE THREAD

**Issue by 12wang3** [exacity/deeplearningbook-chinese#168]
https://github.com/exacity/deeplearningbook-chinese/blob/98daf6b9c2adb9e69e5b62693fad2bb321e703ed/Chapter5/machine_learning_basics.tex#L701
原文:Dividing the dataset into a ﬁxed training set and a ﬁxed test set can be problematic if it results in the test set being small.
应该是“若测试集很小，这将是有问题的”，而不是“若测试集的误差很小”。

**Comment by liber145** [comment]
有道理，多谢指正！
